backend/copr_backend/background_worker_build.py

```diff
@@ -51,6 +58,9 @@
 
 COMMANDS = {
     "rpm_q_builder": "rpm -q copr-rpmbuild --qf \"%{VERSION}\n\"",
+    "echo_authorized_keys": "echo {0} >> /root/.ssh/authorized_keys",
+    "set_expiration": "echo -n {0} > " + USER_SSH_EXPIRATION_PATH,
+    "cat_expiration": "cat {0}".format(USER_SSH_EXPIRATION_PATH),
```

Comment:
I think we should document here that we must be especially careful about error handling; users may replace the `echo` or `cat` commands, and (in theory) cause damage to the backend machine. This deserves documenting IMO (here ore in a later PR).